modules/quickstart-public-cloud/pages/installation.adoc

```diff
@@ -57,6 +57,11 @@ The username and password varies depending on your provider.
 | Microsoft Azure       | ``admin``         |``<instance-name>-suma``
 |===
 
+
+When you log in to the administrator account for the first time, you will have an automatically generated organization name.
+Change this by navigating to menu:Admin[Organizations], and editing the organization name.
+
+
 [IMPORTANT]
 ====
 When you have logged in to the administrator account for the first time, change the default password to protect your account.
```

Comment:
```suggestion
When you have signed in to the administrator account for the first time, change the default password to protect your account.
```
drive-by edit.  Wondering why we first talk about changing the org name and then doing the important password change.